tests/test_dataset.py

```diff
@@ -348,6 +352,41 @@ def setup_dataset(self):
         ]
         self.dataset._data_threshold = 1000000000
 
+    @patched_initialise
+    @patch(ppath + 'MostRecentProc.start')
+    @patch(ppath + 'MostRecentProc.retrieve_entity')
+    @patch(ppath + 'Dataset.lims_ntf', new_callable=PropertyMock)
+    def test_start(self, mocked_lims_ntf, mocked_retrieve_entity, mocked_start, mocked_update):
+        self.dataset.most_recent_proc.entity['status'] = c.DATASET_PROCESSING
```

Comment:
Don't see why we're setting an item here when we're just `del`-ing it two lines later